Sources/Core/ObjectiveCIR.swift

```diff
@@ -246,11 +246,26 @@ public struct ObjCIR {
     }
 
     static func enumStmt(_ enumName: String, body: () -> [String]) -> String {
+        let enums = body()
```

Comment:
We might want to actually have an enum struct where we can query the number of cases in an enumeration vs relying on the actual generated code output

Reply:
Sounds good; where would I put that code? Is there a document describing the design architecture of this tool? It's a very new pattern for me.

Reply:
Please see my update; it's now figuring out the range & type before rendering the ObjC.